tokio/src/runtime/scheduler/current_thread.rs

```diff
@@ -549,14 +536,9 @@ impl Schedule for Arc<Handle> {
                 // Track that a task was scheduled from **outside** of the runtime.
                 self.shared.scheduler_metrics.inc_remote_schedule_count();
 
-                // If the queue is None, then the runtime has shut down. We
-                // don't need to do anything with the notification in that case.
```

Comment:
do we still need to do something to handle the case where the runtime is shutting down?

Reply:
Yes, not sure why CI still passes. I tried adding a new test, but it is a hard case to trigger I guess.